code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -422,7 +422,7 @@
 	if(!attempt_cast(user))
 		return
 
-	for(var/mob/living/M as anything in targets)
+	for(var/mob/living/carbon/M in targets)
```

Comment:
just change the `allowed type` in create_new_targeting

Reply:
allowed type set to mob/living/carbon